src/current/v24.1/known-limitations.md

```diff
@@ -347,6 +347,8 @@ Refer to [`OID` best practices]({% link {{ page.version.version }}/oid.md %}#bes
 
 - Updating subfields of composite types using dot syntax results in a syntax error. [#102984](https://github.com/cockroachdb/cockroach/issues/102984)
 
+- Tuple elements cannot be accessed without enclosing the [composite variable]({% link {{ page.version.version }}/create-type.md %}#create-a-composite-data-type) name in parentheses. For example, `(v).x`. [#114687](https://github.com/cockroachdb/cockroach/issues/114687)
```

Comment:
I'm guessing this will come up most frequently in the context of triggers. Might be good to mention the `OLD` and `NEW` tuples in triggers, or at least link to this from triggers docs.